backend/routes/tours.js

```diff
@@ -191,11 +234,18 @@ router.put('/updateTourDescription', async(request, response, next) => {
         const newSummary = request.body.tour_description;
 
         const results = await request.models.tour.updateTourDescription(tour, newSummary);
-        response.status(200).json(results);
+        if(results){
+            const check = await request.models.tour.getToursbyName(tour);
+            response.status(200).json(check);
+        }
+        else{
+            console.error('There was an error in PUT /tours/updateTourDescription');
+            response.status(400).json('There was an error in PUT /tours/updateTourDescription');
+        }
         next();
     } catch (err) {
             console.error('There was an error in PUT /tours/updateTourDescription', err);
-            response.status(500).json({ message: err.message });
+            response.status(400).json({ message: err.message });
```

Comment:
Leave this one as 500, this is for any error, including server errors (which is 500)